tests/remove_stale_string.py

```diff
@@ -0,0 +1,19 @@
+import re
+import json
+import glob
+from collections import OrderedDict
+
```

Comment:
Btw I don't know where to put this small script as it's not really a test but more some sort of automatic cleaning tool ...